integration-tests/tests/resource-observer/resources-ignored.ejs

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
can we merge the view templates into one, and control using query params?

Reply:
We can but I prefer extra template files.

Reply:
At some point it might make sense but I don't think it does here